lib/reload-client.js

```diff
@@ -0,0 +1,61 @@
+var nanotiming = require('nanotiming')
+var logger = require('nanologger')
+var onIdle = require('on-idle')
+
+if (typeof window !== 'undefined') {
+  document.addEventListener('DOMContentLoaded', function () {
+    onIdle(initReload)
+  })
+}
+
+function initReload () {
+  var timing = nanotiming('bankai:reload')
+  var log = logger('bankai:reload')
+  var source = new window.EventSource('/reload')
+
+  source.addEventListener('open', function () {
+    log.info('connected')
+  })
+
+  source.addEventListener('message', function (event) {
+    try {
+      var ev = JSON.parse(event.data)
+    } catch (e) {
+      return log.error('error parsing event', e)
+    }
+    if (ev.type === 'style') handleStyles()
+    else if (ev.type === 'scripts') handleScripts()
+  }, false)
+  timing()
+
+  function handleScripts () {
+    log.info('scripts', 'reloading')
+    window.location.reload()
```

Comment:
Reloading causes a SSE connection broken error in console, at least in Firefox.
How about adding this?
```js
window.addEventListener('beforeunload', function (event) {
  source.close()
});
```

Reply:
Good call, fixed! :D